.travis.yml

```diff
@@ -12,6 +12,22 @@ matrix:
         - TVOS_SDK=appletvsimulator12.2
         - WATCHOS_SIMULATOR='name=Apple Watch Series 4 - 44mm'
         - WATCHOS_SDK=watchsimulator5.2
+    - os: osx
+      osx_image: xcode10.2
+      env:
+        - CACHE_NAME=SWIFT5_0
+        - SWIFT_VERSION=5.0
+        - IOS_SIMULATOR='name=iPhone 6s,OS=12.2'
+        - IOS_SDK=iphonesimulator12.2
+        - TVOS_SIMULATOR='name=Apple TV 4K (at 1080p)'
+        - TVOS_SDK=appletvsimulator12.2
+        - WATCHOS_SIMULATOR='name=Apple Watch Series 4 - 44mm'
+        - WATCHOS_SDK=watchsimulator5.2    
+      before_install:
+        - rvm install "ruby-2.5.1"
```

Comment:
I think this is the problem, but don't know why yet, it prints `Using /Users/travis/.rvm/gems/ruby-2.5.1` on the output, before print the actual JSON, and this is why it fails to parse it

Reply:
Updated travis config, let's see 😄

Reply:
Now it says to install it, before running swiftlint :D 
`To install do: \'rvm install "ruby-2.5.1"`

Reply:
Yeah, I think that the `.ruby-version` file might be a problem, let's see, just removed it.

Reply:
Could be, I've tried it locally and works
<img width="1002" alt="schermata 2019-02-24 alle 00 09 22" src="https://user-images.githubusercontent.com/17830956/53293255-76585c80-37c8-11e9-8d7a-6ea02c479143.png">